webapp/app.py

```diff
@@ -178,17 +178,39 @@ def _get_contributors():
     return contributors
 
 
-def _filter_contributors(contributors):
-    # Distinguish team_members from contributors
-
-    member_usernames = [member["login"] for member in TEAM_MEMBERS]
+def _filter_team_members_from_contributors(contributors):
+    member_usernames = {member["login"] for member in TEAM_MEMBERS}
     return [
         contributor
         for contributor in contributors
         if contributor["login"] not in member_usernames
     ]
 
 
+def _is_not_renovate_bot(contributor):
+    # Using "login" to check for renovate-bot as
+    # GitHub treats it as a user, i.e., type: "User"
+    
+    return contributor["login"] != "renovate-bot"
```

Comment:
I'd rather not make a function specifically for the `renovate-bot`, and as far as I know this bot is the only special case we need to worry about currently. So for now, can you un-extract this renovate-specific function?

Reply:
`login` can be changed, so this filtering is somewhat [brittle](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-a-user-using-their-id) in the case that renovate decides to change its username. Can we use `id` of `renovate-bot` (found [here](https://api.github.com/users/renovate-bot)) instead?

Reply:
Got it! Using `id` instead of `login`.